lib/node_modules/@stdlib/regexp/to-json/docs/types/index.d.ts

```diff
@@ -0,0 +1,56 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+* JSON representation of a regular expression.
+*/
+interface JSONRepresentation {
+	/**
+	* Value type.
+	*/
+	type: 'RegExp';
+
+	/**
+	* Regular expression pattern.
+	*/
+	pattern: string;
+
+	/**
+	* Regular expression flags.
+	*/
+	flags: string;
+}
+
+/**
+* Returns a JSON representation of a regular expression.
+*
+* @param regexp - regular expression to serialize
+* @returns JSON representation
+*
+* @example
+* var json = toJSON( /ab+c/ );
+* // returns {...}
+*/
+declare function toJSON( regex: RegExp ): JSONRepresentation;
```

Comment:
```suggestion
declare function toJSON( regexp: RegExp ): JSONRepresentation;
```

Variable name should match the JSDoc comment.